locales-pt-BR.xml

```diff
@@ -517,28 +529,32 @@
       <multiple>dirs.</multiple>
     </term>
     <term name="editor" form="short">
-      <single>org.</single>
-      <multiple>orgs.</multiple>
-    </term>
-    <term name="editorial-director" form="short">
-      <single>ed.</single>
-      <multiple>eds.</multiple>
-    </term>
-    <term name="illustrator" form="short">
-      <single>il.</single>
-      <multiple>ils.</multiple>
-    </term>
-    <term name="translator" form="short">
-      <single>trad.</single>
-      <multiple>trads.</multiple>
-    </term>
-    <term name="editortranslator" form="short">
-      <single>ed. e trad.</single>
-      <multiple>eds. e trads.</multiple>
-    </term>
+      <single>org.</single> <!-- no variation for ABNT -->
+      <multiple>org.</multiple> <!-- no variation for ABNT -->
+    </term>
+    <term name="collection-editor" form="short">
+      <single>ed.</single> <!-- no variation for ABNT -->
+      <multiple>ed.</multiple> <!-- no variation for ABNT -->
+    </term>
+      <term name="editorial-director" form="short">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+      <term name="illustrator" form="short">
+        <single>il.</single>
+        <multiple>ils.</multiple>
+      </term>
+      <term name="translator" form="short">
+        <single>trad.</single>
+        <multiple>trads.</multiple>
+      </term>
+      <term name="editortranslator" form="short">
+        <single>ed. e trad.</single>
+        <multiple>eds. e trads.</multiple>
+      </term>
 
     <!-- VERB ROLE FORMS -->
-    <term name="chair" form="verb">chaired by</term>
+    <term name="chair" form="verb">presidido por</term>
     <term name="compiler" form="verb">compiled by</term>
```

Comment:
Not strictly necessary, but could I get you to translate the remaining English verb forms -- I think there's only 10 left or so

Reply:
Yes, thanks! I'm doing it ASAP!

Reply:
I couldn't edit the verb forms from here. Gonna try directly on the file.

Reply:
Guess I figured it out.
I'm still not really sure about the translated verb forms finished by the word "por" (by), because the verbs that come before tend to be on masculine form. I would use the suffix "de", which modifies the verbs to gender neutral forms. But I shall go back to this another time.